src/disease/etl/mondo.py

```diff
@@ -46,7 +46,7 @@ def _construct_dependency_set(self, dag: defaultdict, parent: str) -> set[str]:
 
     _identifiers_url_pattern = r"http://identifiers.org/(.*)/(.*)"
     _lui_patterns: ClassVar = [
-        (NamespacePrefix.OMIMPS, r"https://omim.org/phenotypicSeries/(.*)"),
+        # (NamespacePrefix.OMIMPS, r"https://omim.org/phenotypicSeries/(.*)"),
```

Comment:
this is just commented out for right now to ensure no import errors, but in #197 I need to handle it separately